packages/xstate-react/src/useActorRef.ts

```diff
@@ -84,27 +84,27 @@ export function useActorRef<TLogic extends AnyActorLogic>(
   machine: TLogic,
   ...[options = {}, observerOrListener]: RestParams<TLogic>
 ): ActorRefFrom<TLogic> {
-  const service = useIdleInterpreter(machine, options);
+  const interpretedActor = useIdleInterpreter(machine, options);
```

Comment:
In lots of places we use `actorRef`/`actor` for this already:
```suggestion
  const actorRef = useIdleInterpreter(machine, options);
```